package.json

```diff
@@ -57,9 +59,9 @@
     "mocha": "^10.1.0",
     "office-addin-cli": "^1.5.5",
     "office-addin-debugging": "^5.0.12",
```

Comment:
wasn't this supposed to get updated to 5.0.14?  Looks like I missed this when we were reviewing it together.

Reply:
It's 5.0.12 in the TS version too, so I don't think we missed it. Should it be 5.0.14 in both?

Reply:
One weird thing I noticed is that the 7 new functions I added to the convert script were added to it twice.

Reply:
I guess I was noticing the library version somewhere else.  It would be better to have it current (we've had to make some fixes for sideloading), but we should be getting a new version for the axios fix pretty soon and that would be another version bump.

That wierd about the duplicate functions.

Reply:
OK. Let's wait for the fixed version.